http-clients/src/test/java/org/triplea/http/client/web/socket/WebSocketConnectionTest.java

```diff
@@ -154,52 +199,36 @@ private void verifyErrorHandlerWasCalled() {
       verify(errorHandler).accept(any());
     }
 
-    @Test
-    @DisplayName(
-        "Verify connect failing with exception invokes error handler and pinger is not running")
-    void connectionFailureWithInterruptedException() throws Exception {
-      givenConnectionAttemptThrows();
-
-      final boolean connected =
-          webSocketConnection.connect(webSocketConnectionListener, errorHandler).get();
-
-      assertThat(connected, is(false));
-      verifyPingerNotStarted();
-      verifyErrorHandlerWasCalled();
-    }
-
-    private void givenConnectionAttemptThrows() throws Exception {
-      doThrow(new InterruptedException("test exception"))
-          .when(webSocketClient)
-          .connectBlocking(
-              WebSocketConnection.DEFAULT_CONNECT_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
-    }
-
     @Test
     @DisplayName("Close will close the underlying socket and stops the pinger")
     void close() {
+      webSocketConnection.getWebSocketListener().onOpen(webSocket);
+      requiresCloseAction();
       webSocketConnection.close();
 
-      verify(webSocketClient).close();
+      verify(webSocket)
+          .sendClose(WebSocket.NORMAL_CLOSURE, WebSocketConnection.CLIENT_DISCONNECT_MESSAGE);
```

Comment:
It does not seem to matter what the disconnect message is, or does it? We probably do not need to assert a specific message.

Reply:
It does no longer matter when we don't check for it.
(There's this check that logs an "exception" when the connection is closed server side, which you suggested should be removed)